### PR TITLE
issue #830 completed

### PR DESCRIPTION
**Pull request by tcavaletto**
Currently no link to user search. if you go to /users/id, pulls up page. I made having the url be username a separate task, as that will involve modifying things in the user.rb and I want to research that more.

### CODE REVIEW

File: app/views/search/user_search.html.erb [hunk 1]
```diff
@@ -5,10 +5,14 @@
   <ul>
     	<% for r in @users%>
       		<li>
-          	<strong><%= r.username %></strong> <br> <strong><%= r.name %></strong>
+          	<strong><%= r.username %></strong> 
+             
       		</li>
+          <li class=''>
```
Comment (bammons) [inline app/views/search/user_search.html.erb]: @tcavaletto 
Why is there a class added to this tag and not the one above it?
Also identation needs to be fixed.

File: app/views/search/user_search.html.erb [hunk 2]
```diff
@@ -5,10 +5,14 @@
   <ul>
     	<% for r in @users%>
       		<li>
-          	<strong><%= r.username %></strong> <br> <strong><%= r.name %></strong>
```
Comment (kdorer) [inline app/views/search/user_search.html.erb:8]: Why did you change this? Won't this show the full name on a separate bullet point from the user name?

**Comment by bammons** [comment]
I don't think this completes task #830. 
In my mind requirements for public facing profile are:
-Completely designed and finished layout of profile page
-Contents in page being displayed through database (which looks like for the most part is done)
-And correctly navigable urls for the public user profile (which I think is fair to say that you have completed)

You also have to update your branch from develop, solve merge conflicts, make sure everything still works, and re-push with the contents above + the latest code on develop 

-1 for needing to update branch before merge to develop

**Comment by kdorer** [comment]
-1 get the latest develop first

**Comment by tcavaletto** [comment]
As far as the changes to the username-password display, they were made while testing the search function. The search function now works, so that issue can be closed, and it's much neater than the present code. I've also cleaned up indentation and merged with latest develop. Once I redesign layout I'll update the push again.

**Comment by tcavaletto** [comment]
Code done.

**Comment by bammons** [comment]
+1 when you merge this, follow the process in wiki/pages/git flow

**Comment by seth-marquez** [comment]
+1 to t̶a̶s̶t̶e̶ ̶m̶y̶ ̶r̶a̶i̶n̶b̶o̶w̶

**Comment by kdorer** [comment]
+1
